src/pybind/mgr/dashboard/frontend/src/app/shared/api/smb.service.ts

```diff
@@ -0,0 +1,15 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SmbService {
+  baseURL = 'api/smb/cluster';
+
+  constructor(private http: HttpClient) {}
+
+  list() {
+    return this.http.get(`${this.baseURL}`);
+  }
```

Comment:
since the smb.service is supposed to cover more than just `clusters`, shall we make it like this?
```suggestion
  baseURL = 'api/smb';

  constructor(private http: HttpClient) {}

  listClusters() {
    return this.http.get(`${this.baseURL}/cluster`);
  }
```